frontend/src/index.css

```diff
@@ -11,3 +11,70 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
```

Comment:
Could we use a styled component here?

Reply:
i don't know if i am being old fashioned, but i like the good old className and separate css more than styled components (which i haven't been able to get used to). what do you say?